src/main/target/AOCODARCF4V3/target.h

```diff
@@ -19,8 +19,13 @@
 
 #define USE_TARGET_CONFIG
 
-#define TARGET_BOARD_IDENTIFIER         "AOF4V3"
-#define USBD_PRODUCT_STRING             "AocodaRCF4V3"
+#ifdef AOCODARCF4V3_Flash
+#define TARGET_BOARD_IDENTIFIER "AOF4V3F"
+#define USBD_PRODUCT_STRING     "AocodaRCF4V3_Flash"
+#else
+#define TARGET_BOARD_IDENTIFIER "AOF4V3SD"
+#define USBD_PRODUCT_STRING     "AOCODARCF4V3_SDCard"
```

Comment:
Same issue, please consult company production naming rule.

Reply:
Okay, I've already changed it

Reply:
> Okay, I've already changed it

No, you didn't understand what I mean. Check the above comments.

> We have company naming rule, default for flash version, which is:
> 
>     AOCODARCF4V3 //Flash version
>     AOCODARCF4V3_SD // SDCard version
> 
> Then we need to handle AOCODARCF4V3_SD macron only.,